src/promtail.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright 2023 Canonical Ltd.
+#  See LICENSE file for licensing details.
+
+"""Functions for operating Promtail."""
+
+
+def install() -> None:
+    """Install Promtail."""
+
+
+def config(loki_endpoint: str) -> None:
+    """Configure Promtail.
+
+    Args:
+        loki_endpoint: The Loki endpoint to send logs to.
+    """
```

Comment:
Do you think `install` and `config` could be combined, event, for example, with `start` as well? Or do you think we would potentially want to `stop` and `start` promtail multiple times?

Reply:
Updated accordingly.